app/src/main/java/com/li/weatherapp/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.li.weatherapp.ui.main
+
+import android.content.Context
+import android.content.Intent
+import com.li.weatherapp.R
+import com.li.weatherapp.base.BaseActivity
+
+class MainActivity : BaseActivity() {
+    override val layoutResource: Int
+        get() = R.layout.activity_main
```

Comment:
Tại sao không làm như này ? Có gì khác nhau chăng ? 
```suggestion
    override val layoutResource = R.layout.activity_main
```

Reply:
Nếu dùng get( ) thì chỉ cho phép truyền vào vào layout còn khi không có get() thì có thể truyền giá trị Int bất kỳ